Documentation/Troubleshooting/TYPO3.rst

```diff
@@ -52,7 +52,7 @@ Install Tool Password
 ---------------------
 
 Write access to :file:`config/system/settings.php` (in legacy installations
-:file:`typo3conf/system/settings.php`) is required to reset the
+:file:`typo3conf/ext/LocalConfiguration.php`) is required to reset the
```

Comment:
Since TYPO3 v12 the main configuration file for TYPO3 is `typo3conf/system/settings.php` for non-composer mode (legacy) and `config/system/settings.php`. Thus, this change is not correct.

Until TYPO3 v11 the path was `typo3conf/LocalConfiguration.php` (legacy) or `public/typo3conf/LocalConfiguration.php`.

`typo3conf/ext/LocalConfiguration.php` was never a valid path for the main configuration.

Reply:
@sbuerk Sorry, I thought legacy meant TYPO3 <= 11. I was not aware of non-composer installations in TYPO3 12.
My mistake, however, adding "ext/" in the MR.
You can discard the change.